src/components/WishlistPage/WishlistPage.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import propTypes from 'prop-types';
+
+import WishlistItem from './WishlistItem';
+
+import './WishlistPage.scss';
+
+
+const CN = 'wishlist';
+
+class WishlistPage extends Component {
+  renderProduct(item, action) {
+    return (
+      <WishlistItem key={item.id} item={item} action={action}/>
```

Comment:
You can incapsulate item to action-function here, than you won't need to create new function handleRemoveFromWishlist  in WishlistItem and then WishlistItem can be written as function